### PR TITLE
[WIP] Release/v1.1.0

### PR DESCRIPTION
**Pull request by ubamrein**

### CODE REVIEW

File: dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java [hunk 1]
```diff
@@ -63,7 +63,12 @@ public void upsertExposeesDelayed(List<GaenKey> gaenKeys, OffsetDateTime delayed
 		}
 		var parameterList = new ArrayList<MapSqlParameterSource>();
 		var nowMillis = System.currentTimeMillis();
-		//if delayedReceivedAt is supplied use it
+		// if delayedReceivedAt is supplied use it
+		// else we calculate the last moment of the  current  bucket like this:
```
Comment (ineiti) [inline dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java:67]: Proposition:
```
else calculate what the last millisecond before the next `releaseBucketDuration` will be.
```

Comment (ubamrein) [reply dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java:67]: you mean `the last moment of the current bucket` ;) granted the formulation could be better :D

Comment (ineiti) [reply dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java]: Yes, in fact. Just writing out the formula doesn't tell me _what_ it's supposed to do. And as you see, I got it wrong, even with the formula on multiple lines ;)

In fact I thought about it later and saw my error, but was off for the day...

File: dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java [hunk 2]
```diff
@@ -64,7 +64,8 @@ public void upsertExposeesDelayed(List<GaenKey> gaenKeys, OffsetDateTime delayed
 		var parameterList = new ArrayList<MapSqlParameterSource>();
 		var nowMillis = System.currentTimeMillis();
 		// if delayedReceivedAt is supplied use it
-		// else we calculate the last moment of the  current  bucket like this:
+		// else we calculate the last millisecond of the  current bucket like this (in order to
```
Comment (ineiti) [inline dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java:67]:
```suggestion
		// else we calculate the last millisecond of the current bucket like this (in order to 
```

**Comment by ubamrein** [comment]
TODO: Check that _all_ v1.1.0 changes are included (reverted older PR merge)

**Comment by ubamrein** [comment]
Closing PR due to version conflict

**Pull request closed without merging**